### ISSUE TITLE
Add benchmarks to README

### ISSUE THREAD

**Issue by bartlomieju** [denoland/fastwebsockets#18]

**Comment by nurmohammed840** [comment]
I also did this [**WebSocket Benchmark**](https://github.com/nurmohammed840/web-socket-benchmark)

Result is very interesting!

**Comment by littledivy** [comment]
Hey @nurmohammed840, thanks for sharing the results!

I've opened a PR https://github.com/nurmohammed840/web-socket-benchmark/pull/1 with some fixes

**Comment by nurmohammed840** [comment]
[tokio-tungstenite](https://github.com/snapview/tokio-tungstenite) use internal buffer,

I think we should add `BufReader` to stream (`tokio::io::DuplexStream`) layer? So that a fair comparison can be made...

**Comment by littledivy** [comment]
@nurmohammed840 Yes, it does maintain an internal buffer https://github.com/denoland/fastwebsockets/blob/f3d5c10dc072dbcc3bd98fdca6001e8dc3706685/src/lib.rs#L182 

That is why I removed BufReader in the PR
